test/e2e/lua/dynamic_configuration.go

```diff
@@ -51,8 +51,6 @@ var _ = framework.IngressNginxDescribe("Dynamic Configuration", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(ing).NotTo(BeNil())
 
-		time.Sleep(5 * time.Second)
-
```

Comment:
I totally agree this is really to `sleep` here but the reason I added it here was that I sometimes was getting timeout in the below `WaitForNginxServer` function where it was not able to find ready ingress-nginx-controller pod.